src/app/components/mnoe-subscriptions-list/mnoe-subscriptions-list.html

```diff
@@ -53,22 +53,22 @@
               </td>
               <td>
                 <span style="display: flex;" ng-if="subscription.status !== 'cancelled' && !$ctrl.isAccountManager">
-                  <a ng-click="$ctrl.editSubscription(subscription, 'reactivate')" ng-show="$ctrl.showEditAction(subscription, 'reactivate') && !$ctrl.companyCart" class="btn btn-link btn-link-primary subscription-btn" uib-tooltip="{{ $ctrl.editToolTip('reactivate') | translate }}" tooltip-append-to-body="true">
+                  <a ng-click="$ctrl.editSubscription(subscription, 'reactivate')" ng-show="$ctrl.showEditAction(subscription, 'reactivate') && !$ctrl.companyCart" class="{{ 'btn btn-link btn-link-primary subscription-btn '.concat($ctrl.supportDisabledClass) }}" uib-tooltip="{{ $ctrl.editToolTip('reactivate') | translate }}" tooltip-append-to-body="true">
```

Comment:
couldn't we use ng-class here instead of concat?

Reply:
if so, same for all the other places we're using concat - this doesn't change functionality though so I think this can probably start to be tested as is

Reply:
@adamaziz15 Agreed. I have changed.